lib/rpc/rpc.dart

```diff
@@ -56,6 +58,24 @@ class RPCLive implements RPC {
     return formatted as String;
   }
 
+  @override
+  Future<String> getRefundAddress() async {
+    var address = await _client.call('getnewaddress', ['Sidechain Deposit', 'Sidechain withdrawal refund']) as String;
+    return address;
+  }
+
+  @override
+  Future<double> estimateSidechainFee(int kbyteInWithdrawal) async {
```

Comment:
I think it's fine to just hard core the withdrawal kbyte size within this function